internal/helm/release/manager.go

```diff
@@ -135,6 +136,18 @@ func notFoundErr(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "not found")
 }
 
+func equal(a, b interface{}) bool {
+	if reflect.ValueOf(a).IsNil() || reflect.ValueOf(b).IsNil() {
+		return apiequality.Semantic.DeepEqual(a, b)
+	}
+
+	aBuf, bBuf := &bytes.Buffer{}, &bytes.Buffer{}
+	_ = json.NewEncoder(aBuf).Encode(a)
```

Comment:
It's possible these can error since `a` and `b` can be anything, so I would return errors here.

Reply:
Thank you, you are right